# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nginx/no-integer-tests.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nginx-1.25.3/auto/unix.vanilla	2023-12-23 14:41:04.315664546 +0100
+++ nginx-1.25.3/auto/unix	2023-12-23 14:41:38.827574289 +0100
@@ -638,18 +638,6 @@
 ngx_feature_libs=
 
 
-# C types
-
-ngx_type="int"; . auto/types/sizeof
-
-ngx_type="long"; . auto/types/sizeof
-
-ngx_type="long long"; . auto/types/sizeof
-
-ngx_type="void *"; . auto/types/sizeof; ngx_ptr_size=$ngx_size
-ngx_param=NGX_PTR_SIZE; ngx_value=$ngx_size; . auto/types/value
-
-
 # POSIX types
 
 NGX_INCLUDE_AUTO_CONFIG_H="#include \"ngx_auto_config.h\""
--- nginx-1.25.3/auto/unix.vanilla	2023-12-23 14:50:34.768609611 +0100
+++ nginx-1.25.3/auto/unix	2023-12-23 14:55:11.646823583 +0100
@@ -642,21 +642,6 @@
 
 NGX_INCLUDE_AUTO_CONFIG_H="#include \"ngx_auto_config.h\""
 
-ngx_type="uint32_t"; ngx_types="u_int32_t"; . auto/types/typedef
-ngx_type="uint64_t"; ngx_types="u_int64_t"; . auto/types/typedef
-
-ngx_type="sig_atomic_t"; ngx_types="int"; . auto/types/typedef
-. auto/types/sizeof
-ngx_param=NGX_SIG_ATOMIC_T_SIZE; ngx_value=$ngx_size; . auto/types/value
-
-ngx_type="socklen_t"; ngx_types="int"; . auto/types/typedef
-
-ngx_type="in_addr_t"; ngx_types="uint32_t u_int32_t"; . auto/types/typedef
-
-ngx_type="in_port_t"; ngx_types="u_short"; . auto/types/typedef
-
-ngx_type="rlim_t"; ngx_types="int"; . auto/types/typedef
-
 . auto/types/uintptr_t
 
 . auto/endianness
--- nginx-1.25.3/auto/unix.vanilla	2023-12-27 16:11:54.160513803 +0100
+++ nginx-1.25.3/auto/unix	2023-12-27 16:12:02.789241112 +0100
@@ -646,19 +646,6 @@
 
 . auto/endianness
 
-ngx_type="size_t"; . auto/types/sizeof
-ngx_param=NGX_MAX_SIZE_T_VALUE; ngx_value=$ngx_max_value; . auto/types/value
-ngx_param=NGX_SIZE_T_LEN; ngx_value=$ngx_max_len; . auto/types/value
-
-ngx_type="off_t"; . auto/types/sizeof
-ngx_param=NGX_MAX_OFF_T_VALUE; ngx_value=$ngx_max_value; . auto/types/value
-ngx_param=NGX_OFF_T_LEN; ngx_value=$ngx_max_len; . auto/types/value
-
-ngx_type="time_t"; . auto/types/sizeof
-ngx_param=NGX_TIME_T_SIZE; ngx_value=$ngx_size; . auto/types/value
-ngx_param=NGX_TIME_T_LEN; ngx_value=$ngx_max_len; . auto/types/value
-ngx_param=NGX_MAX_TIME_T_VALUE; ngx_value=$ngx_max_value; . auto/types/value
-
 
 # syscalls, libc calls and some features
 
